src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -168,6 +168,7 @@ export class ZoneServer2016 extends EventEmitter {
   _db!: Db;
   _soloMode = false;
   _useFairPlay = true;
+  _maxPing = 200;
```

Comment:
should be a public property, servers owner would change this

Reply:
> So we don't just lock there weapons we kick them? I didn't saw it that way but i'm ok with it. @Z1Meme What do you think of it?

I've talked with @Z1Meme, hes fine with it